block/manager.go

```diff
@@ -134,6 +136,13 @@ func NewManager(
 		}
 	}
 
+	fcr, ok := fork_choice.GetRule(conf.ForkChoiceRule)
+	if !ok {
+		fcr, ok = fork_choice.GetRule(config.DefaultNodeConfig.BlockManagerConfig.ForkChoiceRule)
```

Comment:
multiple calls to `GetRule` is not needed. when `!ok` simply assign `firstOrderedPure` to `fcr`?